webapp/channels/src/packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -464,32 +464,43 @@ export function fetchChannelsAndMembers(teamId: string): ActionFuncAsync<{channe
     };
 }
 
-export function fetchAllMyTeamsChannelsAndChannelMembersREST(): ActionFuncAsync {
+export function fetchAllMyChannelMembers(): ActionFuncAsync {
     return async (dispatch, getState) => {
         const state = getState();
         const {currentUserId} = state.entities.users;
-        let channels;
+
         let channelsMembers: ChannelMembership[] = [];
         let allMembers = true;
         let page = 0;
-        do {
-            try {
+        try {
+            while (allMembers) {
```

Comment:
Out of scope, but `allMembers` doesn't sound very revealing in this context.

I would go with something more in the lines of `fetchMoreMembers` or `keepFetching` or `hasMoreMembers`.

Reply:
Right - I meant to change that but I forgot :P